GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -105,23 +107,19 @@ public static string GetDefaultHomeDir()
                 return UserHomeDir;
 
             if (EnvUtils.RunningOnWindows())
-            {
                 return WindowsDefaultHomeDir;
-            }
+
             return Environment.GetFolderPath(Environment.SpecialFolder.Personal);
         }
 
         private static string WindowsDefaultHomeDir
         {
             get
             {
-                if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("HOMEDRIVE")))
-                {
-                    string homePath = Environment.GetEnvironmentVariable("HOMEDRIVE");
-                    homePath += Environment.GetEnvironmentVariable("HOMEPATH");
-                    return homePath;
-                }
-                return Environment.GetEnvironmentVariable("USERPROFILE");
+                var homeDrive = Environment.GetEnvironmentVariable("HOMEDRIVE");
```

Comment:
:bulb: Unrelated refactoring is better submitted in a separate pull request.

Reply:
Agree in general. I actually have an open PR #4481 with these sorts of refactorings/tweaks.